src/Controller/BlogController.php

```diff
@@ -120,4 +121,28 @@ public function detailAction(string $articleId, Request $request, SalesChannelCo
             'entry' => $entry,
         ]);
     }
+
+    /**
+     * @HttpCache()
+     * @Route("/blog/rss", name="sas.frontend.blog.rss", methods={"GET"})
+     */
+    public function rss(Request $request, SalesChannelContext $context): Response
+    {
+        $criteria = new Criteria();
+        $criteria->addAssociations(['author.salutation']);
+        $criteria->addFilter(new EqualsFilter('active', 1));
+
+        $results = $this->blogRepository->search($criteria, $context->getContext())->getEntities();
+
+        $page = $this->genericPageLoader->load($request, $context);
+        $page = NavigationPage::createFrom($page);
+
+        $response = $this->renderStorefront('@SasBlogModule/storefront/page/rss.html.twig', [
+            "results"=> $results,
```

Comment:
use `'` instead of `"`. and please fix the space
```suggestion
            'results' => $results,
```